libcontainer/integration/exec_test.go

```diff
@@ -905,3 +905,47 @@ func TestMountCgroupRW(t *testing.T) {
 		}
 	}
 }
+
+func TestPrestartHook(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	root, err := newTestRoot()
+	ok(t, err)
+	defer os.RemoveAll(root)
+
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	config := newTemplateConfig(rootfs)
+	pwd, _ := os.Getwd()
+
+	hookPath := filepath.Join(pwd, "../../script", "hook.py")
+	prestartCmd := configs.Command{Path: hookPath}
```

Comment:
Ah, it looks like this fallback is handled by Go [(1)][1]:

> `Args` holds command line arguments, including the command as `Args[0]`.  If the `Args` field is empty or `nil`, `Run` uses `{Path}`.

So I'm happy with the current args handling as it stands.  I'll file an OCS PR to update the docs there.

[1]: http://golang.org/pkg/os/exec/#Cmd